mne/io/eyelink/_utils.py

```diff
@@ -411,11 +413,30 @@ def _assign_col_names(col_names, df_dict):
         Dictionary of column names for each dataframe.
     """
     for key, df in df_dict.items():
-        if key in ("samples", "blinks", "fixations", "saccades"):
-            df.columns = col_names[key]
-        elif key == "messages":
-            cols = ["time", "offset", "event_msg"]
-            df.columns = cols
+        if key in ("samples", "blinks", "fixations", "saccades", "messages"):
+            cols = col_names[key]
+        else:
+            logger.debug(f"Skipping unsupported EyeLink Event type: {key}")
```

Comment:
Maybe better to make a list of these and at the end of the loop if the list is non-empty `logger.info` it? Depends on how many messages we're skipping and how often I guess...

Reply:
Done in [ace2530](https://github.com/mne-tools/mne-python/pull/13357/commits/ace253083c2e4b409ddf770b2c9436745fb4987f). The number of event types being skipped depends on the file, but typically it might be for example 'BUTTONS' (at least until #13287 is merged). 

I kept the log level at DEBUG for now, I personally don't think the user needs to see this info every time, but if you feel strongly about it I can certainly bump it to INFO.